src/main/java/net/pms/formats/OGG.java

```diff
@@ -27,54 +27,17 @@ public Identifier getIdentifier() {
 		return Identifier.OGG;
 	}
 
-	@Override
-	public boolean transcodable() {
-		return true;
-	}
-
 	/**
 	 * {@inheritDoc}
 	 */
 	@Override
 	public String[] getSupportedExtensions() {
 		return new String[] {
-			"3g2a",
-			"3ga",
-			"3gpa",
-			"3gppa",
-			"aa3",
-			"ac3",
-			"aif",
-			"aiff",
-			"amr",
-			"ape",
-			"at3",
-			"dts",
-			"mka",
-			"mp2",
-			"mpc",
 			"oga",
 			"ogg",
-			"oma",
-			"ra",
-			"shn",
-			"tta",
-			"wv"
+			"oga",
```

Comment:
@Nadahar Isn't "**oga**" already there ?

BTW, i think that **RAM** extension should be added, as it can be played by VLC (AFAIK, the patch for FFMpeg was not merged yet)